.github/workflows/test.yml

```diff
@@ -1,34 +1,35 @@
-name: Go
+# Run basic tests
+
+name: Test
 
 on:
   push:
```

Comment:
```suggestion
```

Reply:
@fmasuhr It would be nice to have actions check every push. If we want to reduce jobs, we can discuss skipping `on: pull_request`. What do you think?